### DISCUSSION TITLE
Censys API

### DISCUSSION THREAD

**Discussion by threatactor666** [projectdiscovery/uncover#48] [Q&A] [answered]
Can you please better explaining how to insert with the correct syntax and maybe with practical example the censys API? I've tried so hard to make it works but it keep not using them. Thanks 
<img width="421" alt="image" src="https://user-images.githubusercontent.com/106819466/177001586-260a16c4-d802-4cdc-a07e-fcf77f4d7af4.png">

**Comment by ehsandeep** [comment, marked as answer]
@threatactor666 that's the right format and working as expected, instead, you can check using `-v` flag to see the exact issue?

<img width="896" alt="image" src="https://user-images.githubusercontent.com/8293321/177002809-d30c7a57-9ece-4589-b668-56e81c063479.png">
